ht6/hw_ig.py

```diff
@@ -0,0 +1,54 @@
+# 1. Implement a function that flatten incoming data:
+# non-iterables and elements from iterables (any nesting depth should be supported)
+# function should return an iterator (generator function)
+# don't use third-party libraries
+
+def merge_elems(*elems):
+    for element in elems:
+        try:
+            iter(element)
```

Comment:
There might be hasattr(element, '__iter__') here also (to avoid try-except block)